connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/MsSqlServerIntegrationSuite.scala

```diff
@@ -97,6 +99,38 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JD
     assert(msg.contains("UpdateColumnNullability is not supported"))
   }
 
+  test("simple scan with LIMIT") {
```

Comment:
shall we move the tests to the parent test suite?

Reply:
OK